Mathlib/Algebra/Polynomial/CoeffMem.lean

```diff
@@ -24,7 +24,7 @@ of `q`.
 namespace Polynomial
 variable {ι R S : Type*} [CommRing R] [Ring S] [Algebra R S]
 
-local notation "deg("p")" => natDegree p
+local notation3 "deg("p")" => natDegree p
```

Comment:
I don't remember the details of this change, it was for the same reason as above.